src/applications/post-911-gib-status/containers/Main.jsx

```diff
@@ -39,7 +39,7 @@ export class Main extends React.Component {
         appContent = genericErrorMessage;
     }
 
-    return <div>{appContent}</div>;
+    return appContent;
   }
```

Comment:
This change is the culprit for 'Main' 'should render' to fail. When I tried rolling this back, the test worked again.

Reply:
In other words, revert `return appContent;` back to `return <div>{appContent}</div>;`

Reply:
Ah, nice! Yeah I can't remember why I removed that.  Will revert.